code/modules/mob/living/simple_animal/hostile/ordeal/white.dm

```diff
@@ -486,8 +487,8 @@
 	maxHealth = 4000
 	health = 4000
 	melee_damage_type = PALE_DAMAGE
-	melee_damage_lower = 35
-	melee_damage_upper = 45
+	melee_damage_lower = 50
+	melee_damage_upper = 60
```

Comment:
Given all the other damage he has, all of which is AoE, i don't think he needs an even stronger melee. He already hits for 70-90% max hp damage, and given we have a whopping 5 armors with Pale 8+ (not including realized, we get +7 with Realized w/o the update), this dude hits like a truck regardless of level. Honestly if you actually want to improve Pale Fixer's fight experience, give him more start-up for his multi-slash. It hits about as hard as the beam but has a 0.4 second startup vs the beam's 1.5 second start up.

Tldr; Raw buffing Pale Fixer's melee damage isn't going to make him or more challenging fight, it's just gonna push people into using guns even more because everything obliterates them in melee regardless of armor.

Reply:
Will consider.